react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class ValidatedInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '', message: 'A password is required.', symbol: 'fas fa-times' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+    if (event.target.value.length === 0) {
+      this.setState({ message: 'A password is required.' });
+      this.setState({ symbol: 'fas fa-times' });
```

Comment:
You can set multiple properties in state with a single setState call. For example:

```JAVASCRIPT
this.setState({
  message: 'A password is required.',
  symbol: 'fas fa-times'
});
```